spec/components/result-view/history-spec.js

```diff
@@ -1,45 +1,38 @@
 "use babel";
 
 import React from "react";
-import Enzyme, { shallow } from "enzyme";
+import Enzyme, { shallow, mount } from "enzyme";
 
 import History from "../../../lib/components/result-view/history";
+import OutputStore from "../../../lib/store/output";
 
 describe("History", () => {
+  let fakeOutput, outputStore, component;
   beforeEach(() => {
     atom.config.set(`Hydrogen.wrapOutput`, false);
+    fakeOutput = {
+      output_type: "display_data",
+      data: {
+        "text/html": "<p>This is some HTML that <b>WILL</b> render</p>",
+        "text/plain": "This is some plain text that WILL NOT render"
+      },
+      metadata: {}
+    };
+    outputStore = new OutputStore();
+    outputStore.index = 0;
+    outputStore.outputs = [fakeOutput, fakeOutput];
+    component = mount(<History store={outputStore} />);
```

Comment:
common practice to call this `wrapper` instead of `component` because its a `ReactWrapper` and contains different properties to it than components.